tests/pre_merge/utils/sweep/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""Test sweep utils."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
```

Comment:
License header inconsistent. 

By the way, what is the correct way of using the Apache-2.0 license header?

Reply:
Good question. I don't know. I can check. I just mindlessly copy headers 🙃

Reply:
According to this https://law.stackexchange.com/questions/61620/is-an-sdpx-header-legally-equivalent-to-a-license-header there is no specific rule to include license headers. Since we use spdx to check, we can stick to its format.

Reply:
Alright, thanks for confirming this. We could stick to the shorter one then :D

Reply:
I have updated the license header